src/main/java/com/upserve/uppend/cli/benchmark/BufferSize.java

```diff
@@ -0,0 +1,17 @@
+package com.upserve.uppend.cli.benchmark;
+
+public enum BufferSize {
+    small(16 * 1024 * 1024 + 16),
```

Comment:
Can you add a comment on the need for `+ 16`?

Reply:
Page size used to include 16 bytes of metadata.
To allow page size == buffer size, I added 16 here.
Removed.